internal/nix/run.go

```diff
@@ -11,9 +11,9 @@ import (
 	"go.jetpack.io/devbox/internal/debug"
 )
 
-func RunScript(nixShellFilePath string, projectDir string, scriptPath string, additionalEnv []string) error {
-	if scriptPath == "" {
-		return errors.New("attempted to run script but did not specify script name")
+func RunScript(nixShellFilePath string, projectDir string, cmdWithArgs string, additionalEnv []string) error {
+	if cmdWithArgs == "" {
+		return errors.New("attempted to run a command but gave an empty command")
```

Comment:
it can be both a command or a script right? so can we say "attempted to run a command or a script but gave an empty one?" or something like this?